templates/home/contextual-signin-home-en.html

```diff
@@ -0,0 +1,193 @@
+---
+{
+	"title": "Home - Canada.ca",
+	"language": "en",
+	"altLangPage": "contextual-signin-home-fr.html",
+	"dateModified": "2022-08-10",
+	"layout": "home",
+	"noReportProblem": true,
+	"auth": {
+		"type": "contextual",
+		"link": "https://www.canada.ca/en/revenue-agency/services/e-services/e-services-individuals/account-individuals.html"
+	}
```

Comment:
Move this config on the main home page example. 

Unless if we are going to apply that new pattern in a far future, please update directly the main home page working example.